### ISSUE TITLE
Button to disable learning mode is missing.

### ISSUE THREAD

**Issue by emelrad12** [exercism/exercism#6585]
![image](https://user-images.githubusercontent.com/16389331/196168961-f1b5424a-d25f-49bb-9e7e-c91389e7e058.png)

as per https://exercism.org/docs/building/product/unlocking-exercises
The button is missing form the ... menu.

**Comment by emelrad12** [comment]
So why cant i access all exercises? is that something else?

**Comment by SleeplessByte** [comment]
Which exercise are you trying to access? Can you search for it here and send us a screenshot? https://exercism.org/tracks/typescript/exercises

**Comment by emelrad12** [comment]
Like any of them 
![image](https://user-images.githubusercontent.com/16389331/196281742-dd2e2b79-c186-4587-aff4-eaa8f00d5294.png)

**Comment by glennj** [comment]
I think that all tracks are configured to lock the exercises until Hello World has been completed.

**Comment by SleeplessByte** [comment]
Yes! You'll always need to complete `hello-world` because it has the instructions how to get the language working on your machine, or the first time you see this language in the web-editor.
